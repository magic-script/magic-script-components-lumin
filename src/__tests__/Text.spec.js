//
import React from "react";
import { create } from "react-test-renderer";
import { Text } from "magic-script-components";

describe("Text component", () => {
  test("Matches the snapshot", () => {
    const text = create(<Text text="Test Text" />);
    expect(text.toJSON()).toMatchSnapshot();
  });
});
