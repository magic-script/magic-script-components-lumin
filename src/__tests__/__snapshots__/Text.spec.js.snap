// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component Matches the snapshot 1`] = `
<text
  text="Test Text"
/>
`;
